### PR TITLE
reversed accidental testnet url change

### PR DESCRIPTION
**Pull request by joelamouche**

### CODE REVIEW

**Comment by joelamouche** [comment]
@jacogr ready for review

File: packages/apps-config/src/settings/ethereumNetworks.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
 // Copyright 2017-2020 @polkadot/apps-config authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-export const ethereumNetworks = ['node-moonbeam', 'moonbeam-alphanet', 'moonbase-alpha', 'moonbeam-standalone'];
+export const ethereumNetworks = ['node-moonbeam', 'moonbase-alphanet', 'moonbeam-standalone'];
```
Comment (jacogr) [inline packages/apps-config/src/settings/ethereumNetworks.ts:4]: It needs `moonbeam-alphanet`, this is what the state.getRuntimeVersion RPC returns -

```
1: state.getRuntimeVersion
{
  specName: moonbeam-alphanet,
  implName: moonbeam-alphanet,
  authoringVersion: 2,
...
```

Comment (joelamouche) [reply packages/apps-config/src/settings/ethereumNetworks.ts:4]: interesting because I get moonbase-alphanet from the stagenet
let me add both then

Comment (joelamouche) [reply packages/apps-config/src/settings/ethereumNetworks.ts]: pushed

**Review by jacogr — APPROVED** [review]

**Pull request merged**